batchlib/analysis/pixel_level_analysis.py

```diff
@@ -142,6 +146,9 @@ def load_sample(self, path):
         return infected, not_infected, serum, background_intensity
 
     def all_stats(self, input_file, output_file):
+        if self.outlier_predicate(input_file):
+            logger.info(f'Excluding outlier from analysis: {input_file}')
+            return
```

Comment:
@constantinpape that might be to naive and cause the input_file not to show in the summary table

Reply:
@Steffen-Wolf should double check and say what the right way to skip images in the analysis is.